app/uk/gov/hmrc/merchandiseinbaggagefrontend/config/AppConfig.scala

```diff
@@ -33,12 +33,24 @@ class AppConfig() extends MongoConfiguration {
   val contactUrl = s"$contactHost/contact/contact-hmrc-unauthenticated?service=$serviceIdentifier"
 }
 
+object AppConfigSource {
+  val configSource: String => ConfigSource = ConfigSource.default.at
+}
+
 trait MongoConfiguration {
   lazy val mongoConf: MongoConf = configSource("mongodb").loadOrThrow[MongoConf]
 }
 
 final case class MongoConf(uri: String, host: String = "localhost", port: Int = 27017, collectionName: String = "declaration")
 
-object AppConfigSource {
-  val configSource: String => ConfigSource = ConfigSource.default.at
-}
\ No newline at end of file
+trait MibConfiguration {
+  lazy val mibConf: MIBConf = configSource("microservice.services.merchandise-in-baggage").loadOrThrow[MIBConf]
+  import mibConf._
+  lazy val mibPort: Int = port
+  lazy val baseUrl: String = s"$protocol://$host:$mibPort"
```

Comment:
mibPort and baseUrl seems to be only used by a test. Move this to the test?

Reply:
is used on line 50 - the purpose is avoiding having hard coded string inside tests too and you can override port for test with wiremock